docs/quadratic/quadratic funding.md

```diff
@@ -0,0 +1,69 @@
+# What Is Quadratic Funding?
+
+Quadratic Funding is a mechanism for allocating funds in a decentralized and democratic manner. It is designed to support public goods, such as open-source software, research, and community initiatives, by leveraging contributions from both large and small donors. The key idea behind Quadratic Funding is to match donations to projects not linearly, but quadratically, amplifying the impact of smaller contributions. [To learn more](https://www.wtfisqf.com/)
+
+1. Fairness and Inclusivity
+ 
+Quadratic Funding aims to create a fair and inclusive funding environment by giving smaller donors a proportionally larger voice. This ensures that projects with widespread support from the community receive adequate funding, regardless of their initial popularity or backing from large donors.
+
+2. Democratization
+
+By enabling anyone to contribute, regardless of the size of their donation, Quadratic Funding democratizes the funding process. It empowers individuals to support projects they believe in, fostering a sense of ownership and engagement within the community.
+
+3. Transparency
+
+Transparency is fundamental to Quadratic Funding. All contributions and allocations are recorded on a public ledger, ensuring accountability and trust in the allocation process. This transparency also enables stakeholders to monitor the distribution of funds and hold administrators accountable for their decisions.
+
+
+# Quadratic Funding Round 
+
+A funding pool (budget) is established to match contributions made to eligible projects. This budget is typically funded by a combination of public and private sources, ensuring sustainability and continued support for public goods. Public goods are voted during the funding round governed by the quadratic-funding contract to ensure funding decisions are made collectively and transparently. Then, votes on proposals are matched using a quadratic formula, where the matching funds increase non-linearly with the number of unique voters. This means that smaller votes receive a higher matching ratio compared to larger ones, amplifying the impact of grassroots support.
+
+
+# Contracts
```

Comment:
This page is intended to explain quadratic funding and the workings of our NFT-based design. It would be best to relocate this section to a new page.